documentation/modules/snip-kafka-versions.adoc

```diff
@@ -10,4 +10,5 @@
 | 3.1.1 | 3.1 | 3.1 | 3.6.3
 | 3.2.0 | 3.2 | 3.2 | 3.6.3
 | 3.2.1 | 3.2 | 3.2 | 3.6.3
+| 3.3.0 | 3.3 | 3.3 | 3.6.3
```

Comment:
does it make sense to start tracking KRaft as well in this table? something like another column called KRaft and has values like "beta, available, GA" or whatever the status is of KRaft in that version.

Reply:
No, I don't think so.